src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -131,13 +131,7 @@ public static function getStyleDefinition($name)
      */
     public function setStyle($name)
     {
-        if ($name instanceof TableStyle) {
-            $this->style = $name;
-        } elseif (isset(self::$styles[$name])) {
-            $this->style = self::$styles[$name];
-        } else {
-            throw new InvalidArgumentException(sprintf('Style "%s" is not defined.', $name));
-        }
+        $this->style = $this->findStyle($name);
```

Comment:
Not sure if it's worth it. At least, given with the current method name (`findStyle()`) I find this barely unreadable. Maybe this gets better when the method name better conveys its function.